app/src/main/java/org/akvo/flow/serialization/response/SurveyedLocaleParser.java

```diff
@@ -73,7 +74,7 @@ public SurveyedLocale parseSurveyedLocale(JSONObject jSurveyedLocale) throws JSO
                 .parseList(jSurveyInstances);
 
         SurveyedLocale surveyedLocale = new SurveyedLocale(id, name, lastModified, surveyGroupId,
-                latitude, longitude);
+                latitude, longitude, SurveyInstanceStatus.SAVED);
```

Comment:
I think I will leave this one as is because this class will be removed in phase 4 as the parsing will be completely refactored